rules/DowngradePhp80/Rector/MethodCall/DowngradeReflectionGetAttributesRector.php

```diff
@@ -80,7 +80,10 @@ public function refactor(Node $node): Ternary|null|int
             return null;
         }
 
-        if (! $this->isObjectType($node->var, new ObjectType('Reflector'))) {
+        if (
+            ! $this->isObjectType($node->var, new ObjectType('Reflector'))
+            && ! $this->isObjectType($node->var, new ObjectType('ReflectionClass'))
```

Comment:
How about directly use ReflectionClass without Reflector?

It seems Reflector doesn't have `getAttributes` method

Reply:
Ok, it seems `getAttributes` introduced in `ReflectionProperty` as well on php 8, so the change seems fine, we can add `ReflectionProperty` when there is use case for it.